appinfo/info.xml

```diff
@@ -26,7 +26,7 @@ Guests users can only access files shared to them and cannot create any files ou
 	<screenshot>https://raw.githubusercontent.com/nextcloud/guests/master/screenshots/settings.png</screenshot>
 	<screenshot>https://raw.githubusercontent.com/nextcloud/guests/master/screenshots/dropdown.png</screenshot>
 	<dependencies>
-		<nextcloud min-version="29" max-version="30" />
+		<nextcloud min-version="30" max-version="30" />
```

Comment:
Since you dropped 29, there is no branch anymore that can be tested against with 29
So I branched off the last commit from master before your PR as `stable29`